### PR TITLE
Create details popup window

### PR DESCRIPTION
**Pull request by hienphan0111**
## Create details popup window
Using javascript to implement flowing project features:

- [ ] When the user clicks (or taps) the button to check project details, the popup with details about the project appears.

- [ ] When the user clicks (or taps) the close (X) button, the popup disappears.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -23,3 +23,123 @@ hbgMenu.addEventListener('click', () => {
...
+    categories: ['Uber', 'Lead Developer', '2015'],
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    featureImage: './image/Snapshoot-Portfolio-3.svg',
+    technologies: ['HTML', 'CSS', 'Javascript', 'github', 'ruby', 'Bootstraps'],
+    liveVersion: '',
+    scource: '',
+  },
```
Comment (HAMMAS-SALEEM) [inline index.js]: inconsistency

- [ ] It's great that you have used array of objects but i found no sequence in project cards. 😨 For example as you can see in the image above ⬆️ that after Tonic Card there is Uber Card but in your array of objects after Tonic card there is Multi-Post Stories 🤔 Kindly fix this issue 👍

Comment (HAMMAS-SALEEM) [inline index.js]: ### Fourth Project Card 💳
Capture

### Modal for Fourth Project Card 📳
modal

- [ ] Now the problem is that if first card and modal for first card is showing correct data according to the array of objects but if i click on second, third or fourth card then modal for these cards don't show the right data 🤔 As you can see the in the above two images that I clicked on the fourth card and popup shows up and the data on the modal is not similar to the card data 🤔 Kindly fix this issue 👍

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @hienphan0111, @Ferizad :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Github Flow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

- [ ] [optional]IT would be nice if you added the Key Features section in your readme file.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**